index.html

```diff
@@ -10,22 +10,14 @@
         </div>
         <div class="col-md-8 col-md-pull-4">
             <p class="introText">
-                <span class="introText--title">Barcelona Free Software</span> is a creators community of Free Software, also known as Open Source, in Barcelona. Our main activity is a <a class="u-fontMono" href="https://gettogether.community/barcelona-free-software/">Get Together</a> / <a class="u-fontMono" href="http://www.meetup.com/{{ site.meetup_username }}/">Meetup group</a>, an open gathering for developers and users who share a love for software freedom.
+                <span class="introText--title">Barcelona Free Software</span> is a creators community of Free Software, also known as Open Source, in Barcelona. Our main activity is a <a class="u-fontMono" href="https://gettogether.community/barcelona-free-software/">Get Together</a> / <a class="u-fontMono" href="http://www.meetup.com/{{ site.meetup_username }}/">Meetup group</a>, an open gathering for developers and users who share a love for software freedom, but sometimes we organize <a href="https://2019.linuxappsummit.org/">bigger events</a>.
```

Comment:
jo posaria, sometimes we organize bigger events (like LInux App Summit 2019) too

Reply:
Actualitzat.